### DISCUSSION TITLE
Containerize Steam with systemd-nspawn

### DISCUSSION THREAD

**Discussion by giscus** [liolok/liolok.com#4] [Comments]
# Containerize Steam with systemd-nspawn

https://liolok.com/containerize-steam-with-systemd-nspawn/

**Comment by WeepingDogel** [comment]
悄悄地坐皓奇哥哥的沙发

**Reply by liolok** [reply]
😼 不可以乱坐！这里是英文版的页面！

**Reply by WeepingDogel** [reply]
略略略！就要就要！！
